components/roadmap-grid/Roadmap.module.css

```diff
@@ -72,6 +73,11 @@
   color: #3182ce;
 }
 
+.item.group div:first-child {
```

Comment:
What is this targeting? Can we add a comment or use an explicit class if one exists?

Reply:
added comment - it is the grouping header and based on the current code not sure of a better selector